core/storage/email/gae_models_test.py

```diff
@@ -128,9 +153,133 @@ def test_get_by_hash_behavior_with_sent_datetime_lower_bound(self):
                     sent_datetime_lower_bound='Not a datetime object')
 
 
+
+class ReplyToIdModelUnitTests(test_utils.GenericTestBase):
+
+
+    """Test the GeneralFeedbackEmailReplyToIdModel class."""
+
+    def setUp(self):
+        super(ReplyToIdModelUnitTests, self).setUp()
+
+        # pylint: disable=unused-argument
+        def _generate_unique_id_for_tests(cls):
+            return 'Non_unique_id'
+
+        self.generate_constant_id_ctx = self.swap(
+            email_models.GeneralFeedbackEmailReplyToIdModel,
+            '_generate_unique_reply_to_id',
+            types.MethodType(
+                _generate_unique_id_for_tests,
+                email_models.GeneralFeedbackEmailReplyToIdModel))
+
+    def test_id_generation_works_correctly(self):
+        #Name is too long for linter.
```

Comment:
Space is required at beginning of comment.
i.e. `# Name is too long for linter`